test/src/Button.transformed.test.tsx

```diff
@@ -18,58 +15,21 @@ export default meta
 
 type Story = StoryObj<typeof meta>
 
+__test(
+  'Primary',
+  testStory('Primary', import.meta.url, composeStories, {
+    include: ['test'],
+    exclude: [],
+    skip: ['skip'],
```

Comment:
The idea was to do the same as the Storybook test-runner, which does:
- skip: show stories in the test results as skipped
- exclude: completely ignore the test, so it won't be shown at all.

However this is quite tricky currently because we'd need to calculate the tags before calling the test function, which we currently don't. I want to revisit this in a subsequent PR.

Reply:
I've never really understood the need for both skip and exclude.  I'd also be in favor of simplifying it.

Reply:
thanks for your input @IanVS! that's good to hear, I'll keep that in mind!